CHANGELOG.md

```diff
@@ -4,9 +4,30 @@ Features:
 
   - DSL metadata updated for Chef 11.18.x and up to Chef 12.4.1.
   ([related issue](https://github.com/acrmp/foodcritic/issues/362)).
+  - Set default `chef_version` to 12.4.1
+  ([related issue](https://github.com/acrmp/foodcritic/pull/364)).
+  - add warnings for use of recommends/suggests.
+  ([related issue](https://github.com/acrmp/foodcritic/pull/309)).
+  - Show progress with --progress
+  ([related issue](https://github.com/acrmp/foodcritic/pull/244)).
+  - Added -l option to list enabled rules and descriptions
+  ([related issue](https://github.com/acrmp/foodcritic/pull/285)).
+  - Fixed `node.force_default` and `node.force_override` along with many other methods on Chef::Node::Attribute that
+  `Chef::Node#method_missing` was forwarding methods to.
+  ([related issue](https://github.com/acrmp/foodcritic/pull/364)).
+  - FC054, Name should match cookbook dir name in metadata
+  ([related issue](https://github.com/acrmp/foodcritic/pull/282))
+  ([related issue](https://github.com/acrmp/foodcritic/pull/358))
+  ([related issue](https://github.com/acrmp/foodcritic/pull/374)).
 
 Bugfixes:
 
+  - merge default options before check instead of during intialization (in Rake task)
+    ([related issue](https://github.com/acrmp/foodcritic/pull/321)).
+  - Fix UTF-8 errors in template checking
+    ([related issue](https://github.com/acrmp/foodcritic/pull/317)).
+  - Make FC041 smarter about CURL usage
+    ([related issue](https://github.com/acrmp/foodcritic/pull/313)).
   - pinned moustache gem to 0.99 for ruby 1.9.3 compatibility
```

Comment:
Side not from the other PR, but can't we unpin this now that 1.9.3 is going away